azureml-minimal/key_vault.tf

```diff
@@ -0,0 +1,14 @@
+data "azurerm_client_config" "cconfig" {}
```

Comment:
You don't need to define another config data source. You can use the one you defined earlier `data.azurerm_client_config.config` in access_policy.tf